cmd/controller-gen/main.go

```diff
@@ -76,9 +76,13 @@ Usage:
 	}
 
 	f := cmd.Flags()
-	f.StringVar(&o.Name, "name", o.Name, "Name to be used as prefix in identifier for manifests")
+	f.StringVar(&o.Name, "name", o.Name, "name to be used as prefix in identifier for manifests")
+	f.StringVar(&o.ServiceAccount, "service-account", o.ServiceAccount, "service account to bind the role to")
+	f.StringVar(&o.Namespace, "service-account-namespace", o.Namespace, "namespace of the service account to bind the role to")
 	f.StringVar(&o.InputDir, "input-dir", o.InputDir, "input directory pointing to Go source files")
-	f.StringVar(&o.OutputDir, "output-dir", o.OutputDir, "output directory where generated manifests will be saved.")
+	f.StringVar(&o.OutputDir, "output-dir", o.OutputDir, "output directory where generated manifests will be saved")
```

Comment:
we should probably lift this up to the root `controller-gen` command.

Reply:
Shouldn't that be another PR?

Reply:
ah, yeah, whoops, you didn't add that, just changes the punctuation.  We can do that in a separate PR.

Reply:
Can you open an issue to track this up-lift? I think you can justify better the reasons behind the suggested change.